index.html

```diff
@@ -119,8 +119,8 @@
 		<h1 id="install-astropy">Install Astropy<a class="paralink" href="#install-astropy" title="Permalink to this headline">¶</a></h1>
         There are a number of ways of installing the latest version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
         <p/><pre>pip install astropy[recommended] --upgrade</pre>
-        If instead you normally use <code>conda</code>, you can do:
-        <p/><pre>conda install -c conda-forge astropy</pre>
+        If instead you normally use <code>conda</code> or <code>miniforge</code>, you can do:
+        <p/><pre>miniforge install astropy</pre>
```

Comment:
I think it is still `conda install`?
```suggestion
        <p/><pre>conda install -c conda-forge astropy</pre>
```